app/pages/organizations/organization-page.jsx

```diff
@@ -183,11 +193,39 @@ class OrganizationPage extends React.Component {
             </div>
             <div className="organization-researcher-words">
               <h4 className="organization-details__heading"><Translate content="organization.home.links" /></h4>
-              <ul>
-                <li className="organization-details__link">Blog</li>
-                <li className="organization-details__link"><i className="fa fa-facebook fa-fw" />- @orgFacebook</li>
-                <li className="organization-details__link"><i className="fa fa-twitter fa-fw" />- @orgTwitter</li>
-              </ul>
+              <div className="organization-details__links">
+                {!!rearrangedLinks.length && rearrangedLinks.map((link, i) => {
```

Comment:
Curious as to why we need the `!!` at the beginning of `!!reaarrangedLinks.length` here. Does the check not work without it?

Reply:
It's a shorthand for writing `Boolean(rearrangedLinks.length)` and explicitly casts an expression to `true/false`. Here are [some examples](https://www.sitepoint.com/javascript-double-negation-trick-trouble/), including a few that regularly trip people up.

Reply:
Slightly related: sometimes you will come across `x = +someVariable` in JavaScript. The plus sign tells the interpreter to explicitly evaluate `someVariable` as a number.

Reply:
Thanks, yeah, I'm familiar with the concept, was more curious as to why it's needed here.

Reply:
That's a really good question. I _think_ it's needed here because there's no explicit `if`, but I would have thought that the `&&` takes care of casting the expressions to Booleans.

Reply:
`if (array.length)` is an implicit check on a truthy or falsy value. `!!` casts it to an explicit boolean. Relying on truthy/falsy values can lead to unexpected behavior: https://www.sitepoint.com/javascript-truthy-falsy/

Using explicit values in conditionals is pretty much always preferred in javascript.

Reply:
FWIW, I tend to always write explicit length checks on arrays in conditionals, `if (array.length > 0)` or `if (array.length > -1)`, etc.

Reply:
I'll change to the explicit length check.
An explicit check or the `!!` solved an issue where if just `rearrangedLinks.length` while it did prevent the `.map` it also returned 0, the length of the empty array, and actually showed the 0 in the DOM.